vector/src/main/AndroidManifest.xml

```diff
@@ -403,10 +399,14 @@
             android:configChanges="orientation|screenSize"
             android:label="@string/settings_notification_privacy" />
         <activity android:name=".activity.VectorWebViewActivity" />
-
         <activity
-            android:name="im.vector.activity.MediaPreviewerActivity"
+            android:name=".activity.MediaPreviewerActivity"
             android:configChanges="orientation|screenSize" />
+        <activity
+            android:name=".activity.KeybackupSetupActivity"
+            android:configChanges="orientation|screenSize"
```

Comment:
I do not like to add configChanges. Is it necessary?

Reply:
removed, was just copy/paste